src/qt/guiutil.cpp

```diff
@@ -944,14 +944,16 @@ QString loadStyleSheet()
     QString styleSheet;
     QSettings settings;
     QString cssName;
+
+    std::set<QString> setValidThemes = {"light","dark","trad"};
```

Comment:
Is there any way to avoid hard coding theme names both here and when they are defined in the Options Dialog? https://github.com/dashpay/dash/blob/e5cd777ec3e671039d89135f69b0747fdfb9845f/src/qt/optionsdialog.cpp#L92-L94

Been looking thru the Qt code but nothing jumps out yet. If we could get a list from `ui->theme->finditems` then we could avoid hard-coding theme names in multiple places.

Reply:
Good point 👍 I force-pushed an alternative solution, pls see 528e249f3